### PR TITLE
Fixed the rescaling bug for robust cost function

### PR DESCRIPTION
**Pull request by fantaosha**
## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have completed my CLA (see **CONTRIBUTING**)
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: theseus/core/robust_cost_function.py [hunk 1]
```diff
@@ -82,7 +82,7 @@ def weighted_jacobians_error(self) -> Tuple[List[torch.Tensor], torch.Tensor]:
         ) = self.cost_function.weighted_jacobians_error()
         squared_norm = torch.sum(weighted_error**2, dim=1, keepdim=True)
         loss_radius = torch.exp(self.log_loss_radius.data)
-        rescale = self.loss.linearize(squared_norm, loss_radius)
+        rescale = (self.loss.linearize(squared_norm, loss_radius) + 1e-20).sqrt()
```
Comment (luisenp) [inline theseus/core/robust_cost_function.py:85]: Does 1E-20 work for `float32` also?

**Review by luisenp — APPROVED** [review]
Small comment, otherwise LGTM!

File: theseus/core/robust_cost_function.py (hunk 1 above)
Comment (fantaosha) [reply theseus/core/robust_cost_function.py:85]: `1e-20` works for `float32` in `torch.autograd.functional.jacobian()`. So I guess it should also work in other cases.

Comment (luisenp) [reply theseus/core/robust_cost_function.py:85]: OK, sounds good, let's merge.

**Pull request merged**